cookbooks/web/templates/default/cgimap.init.erb

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+<%
+pid_directory = node[:web][:pid_directory]
+log_directory = node[:web][:log_directory]
+
+if node[:web][:readonly_database_host]
+  database_host = node[:web][:readonly_database_host]
+  database_readonly = true
+else
+  database_host = node[:web][:database_host]
+  database_readonly = node[:web][:status] == "database_readonly"
+end
+
+memcached_servers = node[:web][:memcached_servers] || []
+%>
```

Comment:
Can we leave all this in the recipe please and just pass the variables in as I don't really like having large chunks of code embedded in templates like this?